docs/runtime/instruments.md

```diff
@@ -32,7 +31,5 @@ Unfortunately, when doing an incremental compilation, only the changed files are
 recompiled and the annotation processor 'forgets' about other instruments that
 haven't been recompiled, leading to runtime errors about missing instruments.
 
-To fix that, we add the
-[`FixInstrumentsGeneration.scala`](../../project/FixInstrumentsGeneration.scala)
-task which detects changes to any of the instruments and forces recompilation of
-all instruments in the project by removing their classfiles.
+To fix that, individual services have to be placed in separate subprojects depending on
+`runtime` and aggregated under `runtime-with-instruments`. Later the META-INF registrations are concatenated in the final uber jar.
```

Comment:
Sorry, my GH suggestion was not properly formatted (as hard to do so in the Web UI) - please run `prettier` on it.